content/docs/operating/configuration.md

```diff
@@ -399,7 +399,7 @@ Prometheus relabeling to control which instances will actually be scraped. Also
 by default all apps will show up as a single job in Prometheus (the one specified
 in the configuration file), which can also be changed using relabeling.
 
-### Zookeeper Serverset SD configurations `<serverset_sd_config>`
+### `<serverset_sd_config>`
```

Comment:
The reason was that the TOC doesn't break and reads a lot cleaner now http://prometheus.io/docs/operating/configuration/